boards: bl654_usb: does not support samples/bluetooth/hci_uart

Hi,
Building hci_uart for bl654_usb board, gives error. (see below)
Samples hci_usb and blinky works for same board.
Sample hci_uart works for ex. board nrf52840dk_nrf52840

Environment:
Version: DTC 1.5.0
Python 3.8.10
cmake version 3.21.1
Zephyr version: 2.6.99
Toolchain: zephyr 0.13.0 

Error log:
-- Configuring done
-- Generating done
-- Build files have been written to: /home/acg/zephyrproject/zephyr/build
-- west build: building application
[1/211] Preparing syscall dependency handling

[23/211] Building C object CMakeFiles/app.dir/src/main.c.obj
FAILED: CMakeFiles/app.dir/src/main.c.obj 
ccache /home/acg/zephyr-sdk-0.13.0/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DBUILD_VERSION=zephyr-v2.6.0-2534-g52b2d39b6a9f -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/home/acg/zephyrproject/zephyr/include -I/home/acg/zephyrproject/zephyr/build/zephyr/include/generated -I/home/acg/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/home/acg/zephyrproject/zephyr/subsys/bluetooth -I/home/acg/zephyrproject/zephyr/subsys/usb -I/home/acg/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/home/acg/zephyrproject/modules/hal/nordic/nrfx -I/home/acg/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/home/acg/zephyrproject/modules/hal/nordic/nrfx/mdk -I/home/acg/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -I/home/acg/zephyrproject/zephyr/modules/hal_nxp/usb/. -I/home/acg/zephyrproject/modules/crypto/tinycrypt/lib/include -isystem /home/acg/zephyrproject/zephyr/lib/libc/minimal/include -isystem /home/acg/zephyr-sdk-0.13.0/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.3.0/include -isystem /home/acg/zephyr-sdk-0.13.0/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.3.0/include-fixed -Os -imacros /home/acg/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -imacros /home/acg/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-address-of-packed-member -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/home/acg/zephyrproject/zephyr/samples/bluetooth/hci_uart=CMAKE_SOURCE_DIR -fmacro-prefix-map=/home/acg/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/home/acg/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT CMakeFiles/app.dir/src/main.c.obj -MF CMakeFiles/app.dir/src/main.c.obj.d -o CMakeFiles/app.dir/src/main.c.obj -c /home/acg/zephyrproject/zephyr/samples/bluetooth/hci_uart/src/main.c
In file included from /home/acg/zephyrproject/zephyr/include/toolchain/gcc.h:66,
                 from /home/acg/zephyrproject/zephyr/include/toolchain.h:43,
                 from /home/acg/zephyrproject/zephyr/include/sys/errno_private.h:10,
                 from /home/acg/zephyrproject/zephyr/lib/libc/minimal/include/errno.h:32,
                 from /home/acg/zephyrproject/zephyr/samples/bluetooth/hci_uart/src/main.c:8:
/home/acg/zephyrproject/zephyr/include/device.h:80:39: error: '__device_dts_ord_DT_CHOSEN_zephyr_bt_c2h_uart_ORD' undeclared here (not in a function)
   80 | #define DEVICE_NAME_GET(name) _CONCAT(__device_, name)
      |                                       ^~~~~~~~~
/home/acg/zephyrproject/zephyr/include/device.h:232:37: note: in expansion of macro 'DEVICE_NAME_GET'
  232 | #define DEVICE_DT_NAME_GET(node_id) DEVICE_NAME_GET(Z_DEVICE_DT_DEV_NAME(node_id))
      |                                     ^~~~~~~~~~~~~~~
/home/acg/zephyrproject/zephyr/include/device.h:246:34: note: in expansion of macro 'DEVICE_DT_NAME_GET'
  246 | #define DEVICE_DT_GET(node_id) (&DEVICE_DT_NAME_GET(node_id))
      |                                  ^~~~~~~~~~~~~~~~~~
/home/acg/zephyrproject/zephyr/samples/bluetooth/hci_uart/src/main.c:34:2: note: in expansion of macro 'DEVICE_DT_GET'
   34 |  DEVICE_DT_GET(DT_CHOSEN(zephyr_bt_c2h_uart));
      |  ^~~~~~~~~~~~~
[28/211] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/heap-validate.c.obj
ninja: build stopped: subcommand failed.

FATAL ERROR: command exited with status 1: /usr/bin/cmake --build /home/acg/zephyrproject/zephyr/build

--- a/boards/arm/bl654_usb/bl654_usb.dts
+++ b/boards/arm/bl654_usb/bl654_usb.dts
@@ -16,6 +16,7 @@
 		zephyr,flash = &flash0;
 		zephyr,console = &bl654_cdc_acm_uart;
 		zephyr,shell-uart = &bl654_cdc_acm_uart;
+		zephyr,bt-c2h-uart = &bl654_cdc_acm_uart;
 		zephyr,code-partition = &slot0_partition;
 	};
 

